### PR TITLE
[doc] Add Markdown component

### PR DESCRIPTION
**Pull request by rwu823**
- 頁面基本樣式
- Markdown Component

@chuanxd @CApopsicle

### CODE REVIEW

File: src/server/render-html.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 module.exports = ({ appHTML, initState = {} })=> {
   return `<!doctype html>
+<link href="//fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i" rel="stylesheet" />
```
Comment (chuanxd) [inline src/server/render-html.js:3]: if need be, Font width has `900, 900i`.

**Review by CApopsicle — APPROVED** [review]
LGTM

**Review by chuanxd — CHANGES_REQUESTED** [review]

File: src/server/render-html.js (hunk 1 above)
Comment (rwu823) [reply src/server/render-html.js:3]: not yet

**Review by chuanxd — APPROVED** [review]

File: src/server/render-html.js (hunk 1 above)
Comment (chuanxd) [reply src/server/render-html.js:3]: hao 8

**Comment by chuanxd** [comment]
This branch has conflicts that must be resolved
@rwu823

File: package.json [hunk 2]
```diff
@@ -64,9 +64,10 @@
     "html-webpack-plugin": "^2.22.0",
     "isomorphic-style-loader": "^1.0.0",
     "jest-cli": "^15.1.1",
-    "node-sass": "^3.10.1",
```
Comment (chuanxd) [inline package.json:67]: GG

**Pull request merged**